playground/assets/index.html

```diff
@@ -266,6 +266,16 @@ <h3>base64</h3>
   inline style
 </p>
 <p class="style-base64-assets">use style class</p>
+<h3>from public folder</h3>
+<style>
+  .style-public-assets {
+    background: url('/icon.png');
```

Comment:
```suggestion
    background: url('/foo/icon.png');
```
`assets` playground has `config.base = '/foo/'`, so I think this needs to be like this.

Reply:
Ah, your right. Sorry for it. :pray: